src/nunjucks/moduk/components/cookie-banner/__tests__/macro.test.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs'
+import { describe, expect, it } from 'vitest'
+
+import { render } from '../../../../../test-utils'
+
+describe('Cookie Banner', async () => {
+  it('renders the content', async () => {
+    const defaultNjk = await fs.promises.readFile(
+      '../__examples__/default.njk',
+      'utf8',
+    )
```

Comment:
Is this ok or shall I be using an inline string?

Reply:
I think it's fine but could be useful to have a `renderFile()` helper for it.